README.md

```diff
@@ -22,16 +22,31 @@ To view the contents of the database, install pgAdmin. Once your account is set
 ## Running database migrations
 **If you are creating a new table, the table name should be singular, i.e. "student" not "students" for consistency.**
 1. Install PostgreSQL and create a database.
+Run ```psql postgres``` to go to the psql shell.
```

Comment:
This does not work for me (I'm using Windows and I have to open psql from the start menu)

Reply:
Yeah sorry, I think I tried installing pgadmin before and I didn't like it so I used homebrew to install. Should I make a separate thing for Mac? I feel like it should be good for Linux too right? (idk for sure tho...)